test/bridge/MixerTest.cpp

```diff
@@ -51,11 +51,11 @@ const bridge::RtpMap FEEDBACK_RTP_MAP = bridge::RtpMap(bridge::RtpMap::Format::R
         if (!(EXPRESSION))                                                                                             \
         {                                                                                                              \
             FAIL() << (std::string()                                                                                   \
-                           .append("Line ")                                                                            \
-                           .append(std::to_string(__LINE__))                                                           \
-                           .append(": expression has failed!. Expression: '")                                          \
-                           .append(#EXPRESSION)                                                                        \
-                           .append("'"));                                                                              \
+                    .append("Line ")                                                                                   \
```

Comment:
Is this due to new clang-format version? They changed something right?

Reply:
not sure. Perhaps it was not properly formatted before